src/schema/objects/suffixes.yaml

```diff
@@ -571,6 +571,11 @@ electrodes:
   display_name: Electrodes
   description: |
     File that gives the location of (i)EEG electrodes.
+ephys:
+  value: ephys
+  display_name: Electrophysiology
```

Comment:
Suggesting this to be more descriptive as well.

Reply:
```suggestion
  display_name: Microelectrode Electrophysiology
```